tools/travis/README.MD

```diff
@@ -1,3 +1,6 @@
-# Info
-
-All scripts utilised by Travis CI should be placed into this folder for the sake of maintainability, and knowing where stuff is
\ No newline at end of file
+# Info
+
+All scripts utilised by Travis CI should be placed into this folder for the sake of maintainability, and knowing where stuff is
+
+# NOTE
```

Comment:
This note was added because when I moved the `dm.sh` script before it broke the servers update pipeline, due to the fact that our current backend relies on this script for updating the server.